src/compiler/output-targets/output-angular.ts

```diff
@@ -175,27 +177,47 @@ export const proxyInputs = (Cmp: any, inputs: string[]) => {
   const Prototype = Cmp.prototype;
   inputs.forEach(item => {
     Object.defineProperty(Prototype, item, {
-      get() { return this.el[item]; },
-      set(val: any) {
-        this.z.runOutsideAngular(() => this.el[item] = val);
+      get() {
+        return this.el[item];
       },
+      set(val: any) {
+        this.z.runOutsideAngular(() => (this.el[item] = val));
+      }
     });
   });
 };
 
 export const proxyMethods = (Cmp: any, methods: string[]) => {
   const Prototype = Cmp.prototype;
   methods.forEach(methodName => {
-    Prototype[methodName] = function() {
+    Prototype[methodName] = function () {
       const args = arguments;
-      return this.z.runOutsideAngular(() => this.el[methodName].apply(this.el, args));
+      return this.z.runOutsideAngular(() =>
+        this.el[methodName].apply(this.el, args)
+      );
     };
   });
 };
 
 export const proxyOutputs = (instance: any, el: any, events: string[]) => {
-  events.forEach(eventName => instance[eventName] = fromEvent(el, eventName));
+  events.forEach(eventName => (instance[eventName] = fromEvent(el, eventName)));
 };
+
+// tslint:disable-next-line: only-arrow-functions
+export function ProxyCmp(opts: { inputs?: any; outputs?: any; methods?: any }) {
```

Comment:
Since ProxyCmp is a private API, we could use arguments instead of a single opts argument, this way it will minify a little better!
```suggestion
export function ProxyCmp(inputs: any, methods?: any) {
```

also, "output" should still be in the constructor (since it applies to own properties)

Reply:
I'm not opposed, but I think the object reads better for debugging capabilities.